### PR TITLE
019 list title

### PR DESCRIPTION
**Pull request by felix-horn**
Please review:
- the list has a title now
- there's haptic feedback with drag and drop
- updated tests
- most importantly: a lot of restructuring for the next big user story: having several lists for different shops

### CODE REVIEW

File: src/02.ShopPage/ShopPage.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components/macro'
+import UncheckedList from './02.Components/UncheckedList'
+import CheckedList from './02.Components/CheckedList'
+import AddItemButton from './01.UI-Elements/AddItemButton'
+import UndoButton from './01.UI-Elements/UndoButton'
+import ShopTitle from './01.UI-Elements/ShopTitle'
+
+export default function ShopPage({
```
Comment (Hecktors) [inline src/02.ShopPage/ShopPage.js:21]: Where are your wonderful prop-types?

File: src/App.js [hunk 2]
```diff
@@ -1,70 +1,43 @@
-import useList from './hooks/useList'
+import { Switch, Route } from 'react-router-dom'
+import useDatabase from './hooks/useDatabase'
 import styled from 'styled-components/macro'
-import UncheckedList from './02.Components/UncheckedList'
-import CheckedList from './02.Components/CheckedList'
-import AddItemButton from './01.UI-Elements/AddItemButton'
-import UndoButton from './01.UI-Elements/UndoButton'
+//import OverviewPage from './01.Pages/OverviewPage'
```
Comment (Hecktors) [inline src/App.js:4]: Did you leave the outcomment line on purpose? 🤔

File: src/App.js [hunk 3]
```diff
@@ -1,70 +1,43 @@
...
-      opacity: 1;
-      pointer-events: all;
-    }
-    100% {
-      opacity: 0;
-      pointer-events: none;
-    }
-  }
-`
+const AppStyled = styled.div``
```
Comment (Hecktors) [inline src/App.js:43]: Far to much styling here. 😜

**Review by Hecktors — COMMENTED** [review]
Sorry, could't find that much. Or congrats? Must be because of the time of the day 😴... or your genius 🤯? Well done, amazing code, world-class tests! Can't imagine buy my Pumuckl-Torte without your app anymore.

File: src/App.js (hunk 2 above)
Comment (dajanaheinze) [reply src/App.js:4]: i wonder the same thing.  Why is it still there?

File: src/App.js [hunk 4]
```diff
@@ -1,70 +1,43 @@
...
-        list={list}
-        handleInputChange={handleInputChange}
-        toggleIsChecked={toggleIsChecked}
-        deleteListItem={deleteListItem}
-      />
-      <UndoButtonStyled className={visibilityUndoButton} onClick={undoDelete} />
-    </>
+    <AppStyled>
+      <Switch>
+        {/* <Route exact path="/">
```
Comment (dajanaheinze) [inline src/App.js:22]: same here? outcomment line

**Review by dajanaheinze — COMMENTED** [review]
looks good :-) two little things

File: src/02.ShopPage/ShopPage.js (hunk 1 above)
Comment (felix-horn) [reply src/02.ShopPage/ShopPage.js:21]: Ooopsiii... :)

File: src/App.js (hunk 2 above)
Comment (felix-horn) [reply src/App.js:4]: All was done in preparation for the next user story. But you're right, I should have left it out. THX

**Pull request merged**